### PR TITLE
Updated a few styles and changed the number in the header

### PR DESCRIPTION
**Pull request by jachurchill**
*Issue #:* /bcgov/entity#6378

*Description of changes:* Updated a few styles and changed the number in the header

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/StaffPayment.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <section id="staff-payment">
     <header>
-      <h2>3. Staff Payment</h2>
+      <h2>4. Staff Payment</h2>
```
Comment (cameron-freshworks) [inline src/components/common/StaffPayment.vue:4]: oh weird that it is picking up a diff here, i believe this change was already in.

**Review by cameron-freshworks — APPROVED** [review]

File: src/components/common/StaffPayment.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/common/StaffPayment.vue:4]: Ditto.

**Review by severinbeauvais — COMMENTED** [review]
1. Looks like this branch needs to be rebased.
2. Should update the app version.
3. Since John has left now, Cam or I will need to pick this up.
4. Since this change is small, Cam or I can probably include this as part of other work.

Thoughts?

**Comment by cameron-freshworks** [comment]
> 1. Looks like this branch needs to be rebased.
> 2. Should update the app version.
> 3. Since John has left now, Cam or I will need to pick this up.
> 4. Since this change is small, Cam or I can probably include this as part of other work.
> 
> Thoughts?

Sorry, late on this reply. Will be picking this up as well, updating the shared components versions and whatnot :)

**Comment by cameron-freshworks** [comment]
Closed this PR. Work was incorporated into pr: https://github.com/bcgov/business-edit-ui/pull/173

**Pull request closed without merging**